support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/ObserverMessage.tsx

```diff
@@ -1,18 +1,39 @@
+import { css } from '@emotion/react';
+import { space } from '@guardian/source/foundations';
 import { ObserverPrint } from 'pages/paper-subscription-landing/helpers/products';
 
+const observerMessageStyle = css`
+	p {
+		margin-bottom: ${space[2]}px;
+	}
+`;
+
 export default function ObserverMessage({
 	observerPrint,
+	startDate,
 }: {
 	observerPrint: ObserverPrint;
+	startDate?: string;
 }) {
-	if (observerPrint === ObserverPrint.SubscriptionCard) {
-		return (
+	const isSubscriptionCard = observerPrint === ObserverPrint.SubscriptionCard;
+	return (
+		<div css={observerMessageStyle}>
+			{isSubscriptionCard && (
+				<p>
+					You should receive your subscription card in 1-2 weeks, but look out
+					for an email landing in your inbox later today containing details of
+					how you can pick up your newspaper before then.
+				</p>
+			)}
 			<p>
-				You should receive your subscription card in 1-2 weeks, but look out for
-				an email landing in your inbox later today containing details of how you
-				can pick up your newspaper before then.
+				{!isSubscriptionCard &&
```

Comment:
Should the `p` tags outside of this be moved inside the `!isSubscriptionCard` condition? Otherwise there will be an empty paragraph in the subscription card case. Or alternatively a single set of p tags outside of both conditions would work (since they're mutually exclusive conditions).